### DISCUSSION TITLE
[@types/chrome] Missing offscreen API

### DISCUSSION THREAD

**Discussion by laplus-sadness** [DefinitelyTyped/DefinitelyTyped#63924] [Issues with a @types package]
Chrome released a new api to create [offscreen documents](https://developer.chrome.com/docs/extensions/reference/offscreen/) (more info [here](https://docs.google.com/document/d/1b-I-vXq2h7OFFmus78jZXIWcKilKJLKLeGplnY9wt7k/edit#heading=h.6pt6jd60gybh)), and the types are not added yet.
Is there a script that auto add the types from their documentation or it's necessary to do it all manually?

[a comment by an automated account is not shown]

**Reply by matcarlson** [reply]
We don't have a script to do that, but I've found a part of the source that could make this easier to manually parse: https://chromium.googlesource.com/chromium/src.git/+/refs/heads/main/extensions/common/api/offscreen.idl

**Comment by laplus-sadness** [comment]
@matcarlson I opened a PR with the changes #63970.
